locations/spiders/coral_travel_pl.py

```diff
@@ -2,24 +2,28 @@
 from scrapy.spiders import CrawlSpider, Rule
 
 from locations.hours import DAYS_PL, OpeningHours
-from locations.structured_data_spider import StructuredDataSpider
+from locations.items import Feature
 
 
-class CoralTravelPLSpider(CrawlSpider, StructuredDataSpider):
+class CoralTravelPLSpider(CrawlSpider):
     name = "coral_travel_pl"
     item_attributes = {"brand": "Coral Travel", "brand_wikidata": "Q58011479"}
-    start_urls = ["https://www.coraltravel.pl/biura-podrozy"]
-    rules = [Rule(LinkExtractor("/biura-podrozy/.+/.+/.+$"), callback="parse_sd")]
+    start_urls = ["https://www.coraltravel.pl/travel-offices"]
+    rules = [Rule(LinkExtractor("/travel-offices/.+/.+$"), callback="parse")]
```

Comment:
As this is a regular expression string, it should be prefixed r""?

Reply:
Fixed in fc895d2.